velox/functions/prestosql/fuzzer/ApproxPercentileResultVerifier.h

```diff
@@ -484,7 +480,7 @@ class ApproxPercentileResultVerifier : public ResultVerifier {
             "multimap_agg(x, w) over ({} {} {}) as bucket",
             getPartitionByClause(groupingKeys_),
             getOrderByClause(sortingKeysAndOrders),
-            getFrameClause(frame))})
+            frameClause)})
```

Comment:
Hi @pramodsatya, could you give me an example where getFrameClause(frame) is different from frameClause and is wrong in this use case? Thanks!